kong-1.4.0-0.rockspec

```diff
@@ -37,14 +38,14 @@ dependencies = {
   "lua-resty-cookie == 0.1.0",
   "lua-resty-mlcache == 2.4.0",
   -- external Kong plugins
-  "kong-plugin-azure-functions ~> 0.4",
-  "kong-plugin-zipkin ~> 0.1",
+  "dndx/kong-plugin-azure-functions dev",
+  "dndx/kong-plugin-zipkin dev",
```

Comment:
rockspec needs to be updated to the released versions

Reply:
Reverted. Thanks!

Reply:
I believe only reverting was not good enough. We also need to bump the versions of these plugins in Kong (so that they don't use the `run_on` field which this PR removes). I will do that.